runtime/src/tests/staking.rs

```diff
@@ -48,19 +51,21 @@ fn stake_and_unstake_with_a_controller_and_stash_as_payee() {
 		));
 
 		// stake is found and usable balance dropped
-		assert_eq!(value, Staking::total_stake(&controller).unwrap());
+		assert_eq!(value, Staking::active_stake(&stash).unwrap());
```

Comment:
@AdaJane , @CAGS295 this doesn't look right to me. Can you elaborate what's the intended behavior.

Here we're changing which value is asserted against while on lines 60-61 below the assertion is against both functions. 

In this example, since that's the first stake being made shouldn't both `total_stake()` and `active_stake()` to be equal to the minimum balance ?

Reply:
FTR adding 

```
assert_eq!(value, Staking::total_stake(&stash).unwrap());
```

will report PASS.

Reply:
re:controller vs stash: iirc the interface changed to accept the stash account instead of the controller account.
To be clear, your proposed change is an additonal assert? (I can't see any reason not to)

Reply:
> re:controller vs stash: iirc the interface changed to accept the stash account instead of the controller account.

I'm arguing about the change from `total_stake()` to `active_stake()` on this particular line. Not about the `controller` -> `stash` change.!

>  To be clear, your proposed change is an additonal assert? (I can't see any reason not to)

Yes, that is correct.